app/screens/MainScreen.tsx

```diff
@@ -13,6 +14,17 @@ import { RootStackParamList } from '../types';
 function MainScreen({
   navigation
 }: StackScreenProps<RootStackParamList, 'Main'>) {
+  // Test database functionality
+  const storage = new Storage();
+  storage.clearStorage();
+  storage.createTable();
+
+  function printLookupResult(codeDescription: String) {
+    console.log(codeDescription);
```

Comment:
Do we want logs in code pushed to main? I see logs as debugging stuff

Reply:
This is in for testing purposes at the moment. :)